client/components/helpCentre/contactUs/UploadFileInput.tsx

```diff
@@ -84,8 +94,13 @@ export const UploadFileInput = (props: UploadFileUploadProps) => {
 				name="imageAttachment"
 				accept={props.allowedFileFormats.join()}
 				multiple={false}
+				ref={inputRef}
 				css={css`
-					display: none;
+					position: absolute;
+					width: 1px;
+					height: 1px;
+					opacity: 0;
+					pointer-events: none;
```

Comment:
Is this change so that we can focus on the file attachment element if it fails validation?